src/pages/ManagePricing/ManagePricing.connector.js

```diff
@@ -0,0 +1,24 @@
+import { graphql } from '@apollo/react-hoc'
+import { flow as compose } from 'lodash'
+import HostPricingQuery from 'graphql/HostPricingQuery.gql'
+import UpdateHostPricingMutation from 'graphql/UpdateHostPricingMutation.gql'
+
+const hostPricing = graphql(HostPricingQuery, {
+  props: ({ data: { hostPricing } }) => ({ hostPricing })
```

Comment:
This will throw if any key here is missing. ` = {}` prevents that ;)

Reply:
eh, we probably want it to throw in that case. Fail early.